### PR TITLE
CS-491: Add new event logging function

### PR DESCRIPTION
**Pull request by TheRaphael0000**
- Add the Other Communication type to the "interaction resume"
- Add a "Log other" feature for res.partner (allow to specify the type, subject, body, direction)

### CODE REVIEW

File: crm_compassion/models/interaction_resume.py [hunk 1]
```diff
@@ -19,13 +19,14 @@ class InteractionResume(models.TransientModel):
...
     communication_type = fields.Selection(
-        [("Paper", "Paper"), ("Phone", "Phone"), ("SMS", "SMS"), ("Email", "Email")]
+        [("Paper", "Paper"), ("Phone", "Phone"), ("SMS", "SMS"), ("Email", "Email"), ("Other", "Other")]
     )
     direction = fields.Selection([("in", "Incoming"), ("out", "Outgoing"), ])
     # Used to display icons in tree view
     state = fields.Selection(related="direction")
     communication_date = fields.Datetime()
     subject = fields.Text()
+    other_type = fields.Text()
```
Comment (ecino) [inline crm_compassion/models/interaction_resume.py]: To be consistent with the type used in the wizard.
```suggestion
    other_type = fields.Char()
```

File: crm_compassion/views/partner_log_other_interaction_wizard_view.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                </sheet>
+                <footer>
+                    <button name="log_interaction" string="Log other" type="object" class="oe_highlight"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <act_window id="action_log_interaction"
+                name="Log other"
```
Comment (ecino) [inline crm_compassion/views/partner_log_other_interaction_wizard_view.xml]:
```suggestion
                name="Log other interaction"
```

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by TheRaphael0000** [comment]
I've committed you suggestion with the GitHub feature, do you still want to me to test these changes on my local setup ?

**Comment by ecino** [comment]
It should be fine, I will test it on stage.

**Pull request merged**